packages/jupyter/src/execute/provider.tsx

```diff
@@ -32,7 +31,6 @@ type ArticleContents = {
   mdast: GenericParent;
   location?: string;
   dependencies?: Dependency[];
-  frontmatter: { thebe?: boolean | Thebe };
```

Comment:
@stevejpurves have removed this type here. We don't use it, and it is now no longer defined on the page - only the project.